src/main/java/uk/gov/hmcts/ccd/domain/service/casedataaccesscontrol/CaseDataAccessControl.java

```diff
@@ -24,4 +28,8 @@ default void grantAccess(String caseId, String idamUserId) {
     CaseAccessMetadata generateAccessMetadata(String caseId);
 
     boolean anyAccessProfileEqualsTo(String caseTypeId, String accessProfile);
+
+    boolean shouldItRemoveCaseDefinitionBaseOnRoleType(Set<AccessProfile> accessProfiles,
```

Comment:
shouldRemoveCaseDefinition --> can we rename to this

Reply:
Yes !! I find extranche this method name? what it would be a good name ?

Reply:
DONE